doc/application/index.rst

```diff
@@ -645,6 +645,29 @@ Zephyr binary into your application directory.
 You can also define the ``SOC_ROOT`` variable in the application
 :file:`CMakeLists.txt` file.
 
+DeviceTree Definitions
+======================
+
+Additional DeviceTree directory tree's, or DTS_ROOT's, can be added by
+creating this directory tree:
+
+    dts/bindings/
+    dts/common/
+    dts/arm/
+    include/
+
+Where 'arm' is changed to the appropriate architecture. Each directory
+is optional. The binding directory contains bindings and the other
+directories contain files that can be included from DT sources.
+
+Once the directory structure is in place, you can use it by specifying
+it's location through the ``DTS_ROOT`` CMake Cache variable::
```

Comment:
it's is a contraction for "it is" (yes, English is inconsistent where possessive is normally indicated by adding "'s" (as in "David's book":
```suggestion
its location through the ``DTS_ROOT`` CMake Cache variable::
```

Reply:
Huh, okay, so it can be possessive for names, but not for 'it'.

Will try to remember, thank you.